src/framework/theme/component/modal/modalPanel.component.tsx

```diff
@@ -14,69 +17,88 @@ export interface ModalPanelProps {
 interface ModalPanelState {
   dialogComponents: Map<string, React.ReactElement<any>>;
   closeOnBackDropAllowed: boolean;
+  modalAnimationConfig: ModalAnimationConfig;
```

Comment:
it could be just `animationConfig`

Reply:
Removed since Modal Service and Panel shouldn't know about passed component animation.